### DISCUSSION TITLE
i'm worried about doing something on a fork

### DISCUSSION THREAD

**Discussion by Endermaster3690** [ScratchAddons/ScratchAddons#2136] [Help] [answered]
might sound stupid to you
so if I get on a fork, make a commit directly to master, but still i am on my fork, will it affect the real scratch addons or just my fork?

also for horizontal block categories i think it is CSS is that correct?

**Comment by mxmou** [comment, marked as answer]
> so if I get on a fork, make a commit directly to master, but still i am on my fork, will it affect the real scratch addons or just my fork?

Just your fork, but there is usually no reason to do that.
